docs/general/contributing.md

```diff
@@ -81,5 +75,89 @@ Wiki locally also adds the benefit of checking for broken links.
 
 !!!info "Run the Wiki Locally"
       For more information about how to run the Wiki locally on your machine visit [the README file on GitHub](https://github.com/w3f/polkadot-wiki/blob/master/README.md).
-      
-This document is based on the [Level contribution guidelines](https://github.com/Level/community/blob/master/CONTRIBUTING.md).
+
+
+
+### General Guidelines
+
+The overarching theme is to ensure the Polkadot Wiki achieves longevity in its content, ensuring that it adheres to the guidelines ensures that maintenance is low and quality of new content is high.
+
+#### Avoid informal language
+
+Avoid using exclamation marks `!` and informal sentences, which are usually more appropriate for blog posts.
+
+#### Grammar and consistency
+
+Running a grammar check before submitting your pull request for review can save reviewers time and speed up the review process.
+
+To ensure consistency across pages, note the following general terms:
+- `relay chain` in text and `Relay chain` at the beginning of a sentence.
+- `parachain` in text and `Parachain` at the beginning of a sentence.
+
+#### Clarity
+
+Prioritize clear, concise, and direct language. Assume the reader has a basic understanding of blockchain concepts but may be new to Polkadot.  
+- **Example:** Instead of: "The utilization of a multi-chain architecture enables the facilitation of diverse transactional modalities."  
+  
+- Write: "Polkadot's heterogeneous, multi-chain design allows for different types of transactions to be validated on the network."  
+
+#### Longevity
+
+Prioritize information that is expected to remain relevant for an extended period. Avoid language that quickly becomes outdated.  
+- **Example:** Instead of: "The upcoming upgrade will introduce new governance features."  
+- Write: "Polkadot governance allows token holders to participate in decision-making." (This describes the general function, not a specific, time-sensitive upgrade)  
+
+#### Accuracy
+
+Ensure all information is technically accurate and up-to-date. The Polkadot ecosystem evolves rapidly, so meticulous attention to detail is crucial.  
+
+!!!note
+    The Polkadot Wiki should only contain features that are live on the Polkadot relay chain, and should not refer to / allude to any upcoming features until they are live.
+
+#### Consistency
+
+Maintain a consistent style and terminology throughout all documentation. This creates a professional and cohesive experience for the reader.  
+- **Example:** Always use the term "parachain" and not "parallel chain" or "para-chain."  
+
+#### Mainnet focus
+
+Document features and information *only* when they are live and available on the Polkadot mainnet. Exclude information about features that are only available on Kusama, Westend, or other testnets.  
```

Comment:
This is duplicated content from the previous banner. We can merge this into the "Longevity" section